poleno/utils/static/poleno/js/collapsable.js

```diff
@@ -0,0 +1,61 @@
+/* Enables elements to automatically show or hide other elements by bootstrap collapsing.
+ * Works on all elements type, which must contain data-toggle="collapse" and
+ * data-target="`css-selector`" attributes. Collapsable element must contain class="collapse".
+ *
+ * data-target-shown elements are shown, when collapsable elements are hidden. data-target-hidden
+ * elements are shown, when collapsable elements are shown and vice versa.
```

Comment:
Nie je to naopak? data-target-shown elements are **shown**, when collapsable elements are **shown**?